File: proj/code-repair/mycode/clean_dataset/patch-single-line/quixbugs-java/codex-sh/lis.java
--- 
+++ 
@@ -16,7 +16,9 @@
 
         if (length == longest || val < arr[ends.get(length+1)]) {
             ends.put(length+1, i);
-            longest = length + 1;
+            if (length+1 > longest) {
+                longest = length+1;
+            }
         }
 
         i++;

File: proj/code-repair/mycode/clean_dataset/patch-single-line/quixbugs-python/codex-sl/lis.py
--- 
+++ 
@@ -10,6 +10,6 @@
 
         if length == longest or val < arr[ends[length + 1]]:
             ends[length + 1] = i
-            longest = length + 1
+            longest = max(longest, length + 1)
 
     return longest
